src/XRP/xrp-error.ts

```diff
@@ -30,6 +31,14 @@ export default class XrpError extends Error {
     'Could not convert payment transaction: (transaction). Please file a bug at https://github.com/xpring-eng/Xpring-JS/issues',
   )
 
+  /**
+   * Encountered a protocol buffer formatted in contradiction to the logic of the XRPL.
```

Comment:
can we link to the docs?

Reply:
good idea.  these'll have to be the generic xrpl.org docs